duckduckgo/iOS
# Add URL to TOSDR matching and use score in calculations

Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/430675284089081

**Description**:
• The TOSDRs are now matched directly with domains rather than guessing a string id from a domain
• The Site rating uses the sign of the terms score to adjust it's own score when a terms classification does not exist
• Removed third party url / tld lookup code as we no longer need this 🎆 

**Steps to test this PR**:
1. Navigate to www.wordpress-com.com and check the Site Ratings logs. Note that the tos is "tos: none_0". The "none" indicates no terms were found. This previously incorrectly matched against the wordpress.com entry.

2. Navigate to www.wordpress.com and note that the logs show "tos: 0_0". This means that the terms were found and the terms score is 0. This did not previously match against the wordpress.com entry.

###### Reviewer Checklist:


###### PR DRI Checklist:
